lib/go/minifier/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/onflow/flow-ft/lib/go/minifier
+
+go 1.14
+
+require github.com/stretchr/testify v1.5.1
+
+replace github.com/onflow/flow-ft/lib/go/minifier => ../minifier
```

Comment:
Why is this needed? It seems to replace itself?

Reply:
all `go.mod` have them e.g https://github.com/onflow/flow-ft/blob/d7d7a0d808b32481f96b34d4c892c184e071c9e1/lib/go/contracts/go.mod#L12. So I followed convention. I believe its for go.mod to use local copy if `minifier` is imported locally.